temoa/extensions/monte_carlo/mc_run.py

```
@@ -199,13 +216,18 @@ class MCRunFactory:
     They will hold the "data tweaks" gathered from input file for application to the base data
     """
 
-    def __init__(self, config: TemoaConfig, data_store: dict):
+    config: TemoaConfig
+    data_store: dict[str, Any]
+    tweak_factory: TweakFactory
+    settings_file: Path
+
+    def __init__(self, config: TemoaConfig, data_store: dict[str, Any]):
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)